### ISSUE TITLE
please re-open and read incorrectly closed bug report #5348 before dismissing

### ISSUE THREAD

**Issue by ikomhoog** [jellyfin/jellyfin#5369]
I reported a bug https://github.com/jellyfin/jellyfin/issues/5348, then there was a pull request attached.
The pull request was finished and closed the issue, but the problem is not solved in the latest unstable release.

Then the previous ticket I made https://github.com/jellyfin/jellyfin/issues/5367 also got closed because of : "Plugin issues belong in the plugin-specific repo"
But getting metadata with the build-in identify button is not a plugin or is it? it's not showing up as a plugin.
I only talked about the plugin because it also had problems and is doing things with TMDb and metadata.
But the main issue of that bug report is about no results when using the identify button.
Also I already made a plugin specific bug report on the plugin page jellyfin/jellyfin-plugin-tmdbboxsets#32

**Comment by cvium** [comment]
Unable to reproduce in master. If you wish for an issue to be re-opened, then write in that issue. Don't create a new one.

The fix is not yet in `stable-rc`. Make sure that you are actually using the latest `unstable`, although I do not recommend using it unless you know what you're doing.

**Comment by ikomhoog** [comment]
"If you wish for an issue to be re-opened, then write in that issue. Don't create a new one." I DID but I cannot re-open it, a collaborator needs to do that.
I commented on it just after you closed it, but I got no response.
![image](https://user-images.githubusercontent.com/9083045/110027850-cad92f80-7d32-11eb-8d81-79557ddb06eb.png)

as per GitHub guidelines, you cannot re-open your own issues if a repo collaborator closed them, since you don't respond on closed tickets I had to make a new ticket to open the old one.
I also replied to the pull request but that also had 0 effect.

Also If you read the issue then you would know that I'm already on the unstable branch and can still reproduce it, hence why I tried reaching out in all the ways.

Also with this message, I just have an option to comment, not to open and comment like I would if I had closed it myself
![image](https://user-images.githubusercontent.com/9083045/110028097-1d1a5080-7d33-11eb-8e41-6e73c1821f5b.png)
